regression/cbmc-library/fprintf-01/test.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+KNOWNBUG
```

Comment:
Any idea why this test is newly broken? I can see the reason for the ones doing integer addresses, but this one seems more surprising.

Reply:
No, that one is puzzling me.

Reply:
Is it a problem that `stdout` and `stderr` are non-deterministic pointers?

Reply:
Probably. Broadly, I see three kinds of failing tests:
* The desired effect of removing the limits on objects and offsets. Test should change or be removed.
* Nondeterminism that now gets resolved in a different way. I.e., the test should be fixed. This one might be in this category.
* The lack of support for the "integer points". Need to think about this one.